Assets/Prefabs/UI/LoadingScreen.prefab

```diff
@@ -146,6 +146,8 @@ MonoBehaviour:
   - If you can't outbid your opponents, the chips you don't spend may come in handy
     later.
   - Cowbots are, contrary to common belief, not cows.
+  - Both you and others can unplug the Tether augment by shooting at the plug, so
```

Comment:
Perhaps "body" instead of "augment" here?

Reply:
Yup